File: OLD Repos/Linear Algebra/Determinant/Determinant Calculation.jl
#### Determinant Calculation

using LinearAlgebra

# Input a n*n array as the determinant you want to calculate.
A = 
[
    4 2 3 5; 
    3 4 6 3;
    2 7 5 4;
    5 3 4 3
]

# Calculate the value of determinant
d = det(A)
println("The value of your determinant is ",d)              # 187
dt = det(A')
println("The value of your transposed determinant is ",dt)  # 187

File: Appendix Algebra/Determinant/Determinant Calculation.py
#### Determinant Calculation

import numpy as np

# Input a n*n array as the determinant you want to calculate.

A = np.array([  [4,2,3,5],
                [3,4,6,3],
                [2,7,5,4],
                [5,3,4,3]   ])

# Calculate the value of determinant
np.linalg.det(A)    # 186.9999999999999
# Calculate the value of transposed determinant
np.linalg.det(A.T)  # 186.9999999999999